contract/src/main.rs

```diff
@@ -1271,206 +1277,353 @@ pub extern "C" fn set_token_metadata() {
-            EntryPointType::Contract,
-        );
+    let collection_name = utils::get_stored_value_with_user_errors::<String>(
+        COLLECTION_NAME,
+        NFTCoreError::MissingCollectionName,
+        NFTCoreError::InvalidCollectionName,
+    );
 
-        // This entrypoint transfers ownership of token from one account to another.
-        // It looks up the owner of the supplied token_id arg. Revert if token is already burnt,
-        // token_id is invalid, or if caller is not owner and not approved operator.
-        // If token id is invalid it reverts with error InvalidTokenID.
-        let transfer = EntryPoint::new(
-            ENTRY_POINT_TRANSFER,
-            vec![
-                Parameter::new(ARG_SOURCE_KEY, CLType::Key),
-                Parameter::new(ARG_TARGET_KEY, CLType::Key),
-            ],
-            CLType::Tuple2([Box::new(CLType::String), Box::new(CLType::Key)]),
-            EntryPointAccess::Public,
-            EntryPointType::Contract,
-        );
+    let new_contract_package_hash_representation =
+        runtime::get_named_arg::<ContractPackageHash>(ARG_NFT_PACKAGE_HASH);
 
-        // This entrypoint approves another account (an operator) to transfer tokens. It reverts
-        // if token_id is invalid, if caller is not the owner, if token has already
-        // been burnt, or if caller tries to approve themselves as an operator.
-        let approve = EntryPoint::new(
-            ENTRY_POINT_APPROVE,
-            vec![Parameter::new(ARG_OPERATOR, CLType::Key)],
-            CLType::Unit,
-            EntryPointAccess::Public,
-            EntryPointType::Contract,
-        );
+    let receipt_uref = get_uref(
+        RECEIPT_NAME,
+        NFTCoreError::MissingReceiptName,
+        NFTCoreError::InvalidReceiptName,
+    );
 
-        let set_approval_for_all = EntryPoint::new(
-            ENTRY_POINT_SET_APPROVE_FOR_ALL,
-            vec![
-                Parameter::new(ARG_TOKEN_OWNER, CLType::Key),
-                Parameter::new(ARG_APPROVE_ALL, CLType::Bool),
-                Parameter::new(ARG_OPERATOR, CLType::Key),
-            ],
-            CLType::Unit,
-            EntryPointAccess::Public,
-            EntryPointType::Contract,
-        );
+    let new_receipt_string_representation = format!(
+        "nft-{}-{}",
+        collection_name,
+        new_contract_package_hash_representation.to_formatted_string()
+    );
+    storage::write(receipt_uref, new_receipt_string_representation);
 
-        // This entrypoint returns the token owner given a token_id. It reverts if token_id
-        // is invalid. A burnt token still has an associated owner.
-        let owner_of = EntryPoint::new(
-            ENTRY_POINT_OWNER_OF,
-            vec![],
-            CLType::Key,
-            EntryPointAccess::Public,
-            EntryPointType::Contract,
-        );
+    match identifier {
+        NFTIdentifierMode::Ordinal => utils::break_up_owned_tokens_in_ordinal_mode(),
```

Comment:
in all places using `break_up_` naming convention, change to `migrate_`

Reply:
Changed to migrate